File: Naloga 30/naloga_30.cpp
#include <iostream>

void naloga_30 () {
    int x, y=0;
    std::cin >> x;
    if (x < 1) {
        y = x*10;
    }
    else {
        y = x;
    }
    std::cout << y << std::endl;
};

int main () {
    naloga_30();
    return 0;
};

File: Naloga 30/naloga_30.py
def naloga_30():
    x = input("vnesi stevilo")
    y = 0
    if int(x) < 1:
        y = x*10
    else:
        y = int(x)
    print(y)
    return
if __name__ == '__main__':
    naloga_30()
